operator/src/test/java/oracle/kubernetes/operator/helpers/DomainValidationStepTest.java

```diff
@@ -25,10 +26,12 @@
 import oracle.kubernetes.utils.TestUtils;
 import oracle.kubernetes.weblogic.domain.model.Configuration;
 import oracle.kubernetes.weblogic.domain.model.DomainCondition;
+import oracle.kubernetes.weblogic.domain.model.DomainFailureSeverity;
 import oracle.kubernetes.weblogic.domain.model.DomainResource;
 import oracle.kubernetes.weblogic.domain.model.DomainStatus;
 import oracle.kubernetes.weblogic.domain.model.ManagedServer;
 import oracle.kubernetes.weblogic.domain.model.Model;
+import org.jetbrains.annotations.NotNull;
```

Comment:
We've been using javax.annotation.NonNull, rather than a Jetbrains annotation.

Reply:
Please note that this change is in the unit test code. I don't see use of javax.annotation.NotNull in unit test code.

Here are some examples in the other unit test classes that use org.jetbrains.annotations.NotNull.
helpers/RollingHelperTest.java:import org.jetbrains.annotations.NotNull;
helpers/TopologyValidationStepTest.java:import org.jetbrains.annotations.NotNull;
makeright/FailureReportingTest.java:import org.jetbrains.annotations.NotNull;
steps/InitializeWebhookIdentityStepTest.java:import org.jetbrains.annotations.NotNull;